index.js

```diff
@@ -9,20 +9,12 @@ import express from 'express';
 import hathorLib from '@hathor/wallet-lib';
```

Comment:
I think we should change to `import { Wallet } from @hathor/wallet-lib`.

Reply:
Done!